tests/e2e/features/artifacts/whereToSendPage/artifacts.feature

```diff
@@ -0,0 +1,80 @@
+@artifacts @regression @whereToSendPage @emptyWallet @various
+Feature: Artifacts - UI
+
+  Background:
+    Given Connect Metamask extension with login action
+
+  @id1560
```

Comment:
Please add tags for mainnet and testnet networks if any. (as you use separation in the tests)